kview/src/main/java/dev/ikm/komet/kview/mvvm/view/pattern/PatternDetailsController.java

```diff
@@ -729,14 +740,18 @@ private void savePattern(ActionEvent actionEvent) {
         boolean isValidSave = patternViewModel.createPattern();
         LOG.info(isValidSave ? "success" : "failed");
         updatePatternBanner();
+        updateView();
     }
 
     private void updatePatternBanner() {
         // update the title to the FQN
         patternTitleText.setText(patternViewModel.getPatternTitle());
 
         // get the latest stamp time
-        updateUIStamp(getStampViewModel());
+        if (patternViewModel.isPatternPopulated()) {
+            updateTimeText(getStampViewModel().getValue(TIME));
+        }
+       // updateUIStamp(getStampViewModel());
```

Comment:
we can probably delete this commented out line

Reply:
Done